src/Rectangle.js

```diff
@@ -129,6 +129,32 @@ class Rectangle {
     get height () {
         return Math.abs(this.top - this.bottom);
     }
+
+    points (vec) {
```

Comment:
Will add comments here...

Basically I wrote this without using generator functions because `regeneratorRuntime` is kinda a big dependency.